lib/ansible/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -442,11 +442,15 @@ def install(self):
                 self.module.fail_json(
                     msg="Jenkins home directory doesn't exist.")
 
-            md5sum_old = None
-            if os.path.isfile(plugin_file):
-                # Make the checksum of the currently installed plugin
-                md5sum_old = hashlib.md5(
-                    open(plugin_file, 'rb').read()).hexdigest()
+            try:
+                sha1_old = hashlib.sha1(open(plugin_file, 'rb').read())
+            except Exception:
+                e = get_exception()
```

Comment:
get_exception is deprecated API (used for Python 2.4). Could you please use native Python syntax for exception handling `except Exception as e`